iocore/eventsystem/P_UnixEventProcessor.h

```diff
@@ -36,6 +36,7 @@ EventProcessor::EventProcessor() : n_ethreads(0), n_thread_groups(0), n_dthreads
   memset(all_dthreads, 0, sizeof(all_dthreads));
   memset(n_threads_for_type, 0, sizeof(n_threads_for_type));
   memset(next_thread_for_type, 0, sizeof(next_thread_for_type));
+  ink_mutex_init(&dedicated_spawn_thread_mutex, "EventProcessorDedicateThreadSpawn");
```

Comment:
Doesn't matter - I have another PR stacked after this that will eliminate the name argument for `ink_mutex_init` because it's useless. The argument is simply ignored. But I need this fix first because all the other PRs break. But, since I have to clang format again, I'll fix it.